app/javascript/utils/form/final_form/SearchField.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Field } from 'react-final-form'
+import classNames from 'classnames'
+import { Label, FieldError, HelpText } from './Field'
+
+export const SearchField = ({
```

Comment:
I think this component should use `export default` since the file is only exporting one component